### DISCUSSION TITLE
Can rectangle send apps to left/right monitor?

### DISCUSSION THREAD

**Discussion by adriangalilea** [rxhanson/Rectangle#1384] [Q&A] [answered]
I haven't seen it in the features, but figured I'd ask in case either I missed it or it's coming eventually.

**Comment by rxhanson** [comment, marked as answer]
Yep, it's in there.
![image](https://github.com/rxhanson/Rectangle/assets/13651296/1cb5924c-1b35-4eb2-a311-aec278c5ea3c)

**Reply by adriangalilea** [reply]
I completely missed it sorry, works flawless, I just deactivated btt snapping in favor of this, loving it. Thanks!
